student.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+
+# Creating class Student
+class Student < Person
+  def initialize(age, classroom, name: 'Unknown', parent_permission: true)
+    super(name, age, parent_permission)
+    @classromm = classroom
+  end
+
+  super(name, age, parent_permission)
```

Comment:
Can you explain what this line is for here? The super(name, age, parent_permission) line outside of any method   seems to be a misplaced duplication of the constructor call.

Reply:
Hey @souad988, sorry is a typo error! I will remove it. Thank you for the review!